src/main/pages/setup-openshift/openshift-single-user.md

```diff
@@ -142,6 +142,24 @@ oc apply -f che-server-pvc.yaml
 oc set volume dc/che --add -m /data --name=che-data-volume --claim-name=che-data-volume
 ```
 
+## HTTPS Mode
+
+<span style="color:red;">IMPORTANT!</span> Self-signed certificates aren't acceptable.
```

Comment:
I don't understand what this means - you say it's unacceptable, but then you seem to redirect to instructions on how to do it...

Reply:
Yeah, makes sense. Fixed.